exercises/hamming/example.js

```diff
@@ -1,24 +1,17 @@
-class Hamming {
-
-  compute(strand1, strand2) {
-    const len1 = strand1.length;
-    const len2 = strand2.length;
-    if (len1 !== len2) {
-      throw new Error('left and right strands must be of equal length');
-    }
+export const compute = (strand1, strand2) => {
+  const len1 = strand1.length;
+  const len2 = strand2.length;
+  if (len1 !== len2) {
+    throw new Error('left and right strands must be of equal length');
+  }
 
-    let distance = 0;
-    let idx = -1;
-    const end = len1; // there could be len2, they're equal
-    while (++idx < end) {
-      if (strand1[idx] !== strand2[idx]) {
-        distance++;
-      }
+  let distance = 0;
+  const end = len1; // there could be len2, they're equal
+  for (let idx = 0; idx < end; idx += 1) {
+    if (strand1[idx] !== strand2[idx]) {
+      distance += 1;
     }
-
-    return distance;
   }
 
-}
-
-export default Hamming;
+  return distance;
```

Comment:
Using reduce should remove declaring extra variable `distance`. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce

Like `return array.reduce()`

I am OK with merging this as is, but would favour the `reduce` version. Would you mind changing it?